node-require/calculate.js

```diff
@@ -0,0 +1,23 @@
+const add = require('./add');
+const subtract = require('./subtract');
+const multiply = require('./multiply');
+const divide = require('./divide');
+
+const firstNumber = parseInt(process.argv[2]);
+const secondNumber = parseInt(process.argv[4]);
```

Comment:
`calculate.js` should still produce accurate calculations if the user enters non-integer numbers.